docs/source/infer.reparam.rst

```diff
@@ -50,6 +50,24 @@ Discrete Cosine Transform
     :special-members: __call__
     :show-inheritance:
 
+Haar Transform
```

Comment:
is this the right heading?

Reply:
Yes it matches the `Discrete Cosine Transform` heading above. The headings implicitly assume reparameterization.